TAScheduler/models.py

```diff
@@ -18,6 +18,7 @@ def __str__(self):
 class TA(models.Model):
     user = models.ForeignKey(User, on_delete=models.CASCADE, null=False)
     grader_status = models.BooleanField()
+    skills = models.CharField(max_length=30,default="idk why this is needed")
```

Comment:
This should be a Django Textfield. That will take a large input of text and not have a maximum length